### DISCUSSION TITLE
Please include foryoufeed on pachli

### DISCUSSION THREAD

**Discussion by silicology1** [pachli/pachli-android#503] [Ideas]
Chronological order of feed is overwhelming to view. Another tab for personalized feed would be great. 

Here is the javascript package: 

https://github.com/pkreissel/fedialgo

https://github.com/pkreissel/foryoufeed

Website:
https://foryoufeed.vercel.app/

<img width="603" alt="feed-algorithm" src="https://github.com/pachli/pachli-android/assets/2051004/5af61555-a674-4ceb-b415-b6271bc9ebf8">

**Comment by nikclayton** [comment]
Hi, thanks for the suggestion.

That's not something that's on the roadmap at the moment, I'm still focused on trying to ensure that all the features your server provides is supported.

I am working on providing general controls on what's in each timeline that you can use everywhere (hide boosts, only posts with media, that sort of thing) but it wouldn't add extra content to the feed your server provides, only filter it in some way.

There are other services you might be interested in, like https://newsmast.org/. At the moment you have to use their client, but I have contacted them to ask if supporting the Mastodon API, so clients like Pachli can treat them like any other Mastodon server.

As I'm not sure this is actionable at the moment I'm going to transfer it from an issue to a discussion, where other people can add any recommendations they have for additional services that can be used to change the contents of your home timeline.

**Comment by michelcrypt4d4mus** [comment]
i've done quite a lot of work on @pkreissel's `fedilago` implementation. Added new weightings/scorings, live filtering based on hashtags, language, etc., and a bunch of other stuff. 

* Video of it in action: https://universeodon.com/@cryptadamist/113601673411306020
* Fork of `fedialgo`: https://github.com/michelcrypt4d4mus/fedialgo
* Fork of the demo app: https://github.com/michelcrypt4d4mus/fedialgo_demo_app_foryoufeed
